docs/methods/search_parameter_optimization.md

```diff
@@ -0,0 +1,131 @@
+# Search Parameter Optimization
+
+## Calibration and optimization
+### Overall process
+The first step of every alphaDIA run is the optimization of search parameters and the calibration of the empirical or fully predicted spectral library to the observed values. This step has two main purposes: 1) removing the systematic deviation of observed and library values, and 2) optimizing the size of the search space to reflect the expected deviation from the library values. For DIA search this means calibration and optimization of certain parameters: retention time, ion mobility, precursor m/z and fragment m/z. The process of iterative calibration and optimization is illustrated below.
+
+<img src="../_static/images/methods_optimization.png" width="100%" height="auto">
+
+Optimization can be performed either in either a targeted or automatic manner. In targeted optimization, the search space is progressively narrowed until a target tolerance is reached for a given parameter. In automatic optimization, the search space is progressively narrowed until an internal algorithm detects that further narrowing will reduce performance, at which point the optimal value is selected for search. Automatic optimization can be triggered by setting the target tolerance to 0.0 in the GUI, or by setting it to any nonpositive value in the command line. It is possible to use targeted optimization for some parameters and automatic optimization for others; currently, it is recommended to use targeted optimization for precursor m/z, fragment m/z and ion mobility, and automatic optimization for retention time.
```

Comment:
"performance" -> what is meant here? speed? quality?

Reply:
I've rephrased it.